app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -121,7 +121,7 @@ def use_v4=(value)
     end
 
     def use_v4
-      self.attrs[:ovirt_use_v4] || false
+      self.attrs[:ovirt_use_v4] || true
```

Comment:
This effectively makes it impossible to use v3, right? It's effectively the same as:
```ruby
def use_v4
    true
end
```

I think this should work:
```suggestion
      self.attrs.fetch(:ovirt_use_v4, true)
```

Reply:
Got It, Thanks! :)